src/mono/browser/browser.proj

```diff
@@ -332,8 +332,8 @@
       { "identity": "WasmNativeStrip",              "defaultValueInRuntimePack": "true" },
       { "identity": "WasmSingleFileBundle",         "defaultValueInRuntimePack": "false" },
       { "identity": "WasmEnableSIMD",               "defaultValueInRuntimePack": "true" },
-      { "identity": "WasmEnableExceptionHandling",  "defaultValueInRuntimePack": "true" },
-    ],
+      { "identity": "WasmEnableExceptionHandling",  "defaultValueInRuntimePack": "true" }
```

Comment:
Could we generate the JSON from actual property values?

Reply:
Why do we need actual property values? How do we extract runtime pack actual property values?
- In this place only `WasmEnableExceptionHandling` and `WasmEnableSIMD` are populated.
- Do the defaults change in time?